server/chat-commands.js

```diff
@@ -3740,6 +3740,40 @@ const commands = {
 	},
 	importinputloghelp: [`/importinputlog [inputlog] - Starts a battle with a given inputlog. Requires: + % @ & ~`],
 
+	acceptdraw: 'acceptdraw',
+	offerdraw(target, room, user, connection, cmd) {
+		const battle = room.battle;
+		if (!battle) return this.errorReply("Must be in a battle room.");
+		if (!Config.allowrequestingties) return this.errorReply("This server does not allow the offering/accpeting of draws");
```

Comment:
Typo in `accpeting`, but I'm not a huge fan of this phrasing, anyway.

```suggestion
		if (!Config.allowrequestingties) return this.errorReply("This server does not allow offering draws.");
```